mozdef_util/mozdef_util/elasticsearch_client.py

```diff
@@ -38,6 +38,12 @@ def __init__(self, servers, bulk_amount=100, bulk_refresh_time=30):
         self.es_connection.ping()
         self.bulk_queue = BulkQueue(self, threshold=bulk_amount, flush_time=bulk_refresh_time)
 
+    def close_index(self, index_name)
```

Comment:
This is a syntax error, you need a semicolon at the end of the line.

Reply:
fixed in last commit.